lib/blocks.js

```diff
@@ -5,11 +5,17 @@ var async = require('async');
 var bitcore = require('bitcore-lib');
 var pools = require('../pools.json');
 var BN = bitcore.crypto.BN;
+var LRU = require('lru-cache');
+
 
 function BlockController(node) {
   var self = this;
   this.node = node;
 
+  this.blockSummaryCache = LRU(1000000);
+  this.blockCacheConfirmations = 6;
+  this.blockCache = LRU(1000);
```

Comment:
We may have discussed this before, but I just forgot. Was there any reasoning behind why we picked these particular numbers for the cache size, or did they just seem like reasonable numbers? Are there any particular memory usage constraints that we want to meet?

Some users running insight-api on systems without much RAM might want to be able to tweak these numbers if the LRU cache takes up more memory than they like. Would it be easy to make these configurable? We could at least take the values in as arguments from the constructor.

lru-cache can take in maxAge as an option. Allowing that to be set could offer another way to go for our raspberry-pi users.